renderer/csv.go

```diff
@@ -49,12 +49,12 @@ func RenderCSV(request *models.RenderRequest) ([]byte, error) {
 func writeTitles(writer *csv.Writer, request *models.RenderRequest) error {
 	err := writeRow(writer, request.Title)
 	if err != nil {
-		log.ErrorC(request.Filename, err, log.Data{"_message": "Unable to write title to csv", "value": request.Title})
+		log.Event(nil, fmt.Sprintf("unable to write title to csv: %s", request.Title), log.ERROR, log.Error(err))
```

Comment:
same comment as in models.go (instead of `fmt.Sprintf(...)` we should use `log.Data{}` to provide the values) - also for the other logs in this file